src/jqLite.js

```diff
@@ -671,7 +671,10 @@ forEach({
       }
       return element.value;
     }
-    element.value = value;
+    // Workaround for Firefox validation #12102.
+    if (element.value != value) {
```

Comment:
`element.value` is always a string. If the value we are going to set it to will be transformed to the same string, then do not copy the value